migrations/dlu/12_modular_build_ugc.sql

```diff
@@ -1,5 +1,4 @@
 CREATE TABLE IF NOT EXISTS ugc_modular_build (
```

Comment:
having the character_id as a FK on this table would help remove data when characters are deleted.
sooo something like...

```sql
CREATE TABLE IF NOT EXISTS ugc_modular_build (
	ugc_id BIGINT NOT NULL PRIMARY KEY,
	character_id BIGINT NOT NULL,
	ldf_config VARCHAR(60) NOT NULL,
	FOREIGN_KEY (character_id) REFERENCES charinfo(id) ON DELETE CASCADE,
);
```

Additionally, when entering modular build mode with a vehicle, is this data entry cleaned up? or is it reused somehow?